services/movie.py

```diff
@@ -0,0 +1,34 @@
+from django.db.models import QuerySet
+
+from db.models import Movie
+
+
+def get_movies(
+        genres_ids: list[int] = None,
+        actors_ids: list[int] = None
+)-> QuerySet | Movie:
```

Comment:
The return type annotation `QuerySet | Movie` is incorrect. Since `get_movies` always returns a `QuerySet`, it should be `QuerySet[Movie]` instead.